From d6be17b5d00a1d601492d30e2fe8cc30fbd4152d Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 6 Nov 2024 10:17:38 -0800
Subject: [PATCH] drivers_py: Update systemd service for Yocto

---
 config/systemd/oasis_drivers.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/systemd/oasis_drivers.service b/config/systemd/oasis_drivers.service
index 54ca67a..1364c94 100644
--- a/config/systemd/oasis_drivers.service
+++ b/config/systemd/oasis_drivers.service
@@ -11,7 +11,7 @@ After=network.target
 # TODO: Better way to define this
 #
 
-User=garrett
+User=root
 
 #
 # ROS 2 configuration
@@ -24,7 +24,7 @@ Environment=RMW_IMPLEMENTATION=rmw_fastrtps_dynamic_cpp
 # Path configuration
 #
 
-Environment=OASIS_ENV=/home/${USER}/Documents/ros-ws/src/oasis/ros-ws/oasis-${ROS2_DISTRO}/install/setup.bash
+Environment=OASIS_ENV=/opt/ros/${ROS2_DISTRO}/setup.bash
 
 #
 # Service definition
-- 
2.34.1

